libparsec/crates/crypto/src/rustcrypto/sequester.rs

```diff
@@ -148,7 +149,7 @@ impl SequesterPublicKeyDer {
     // Encryption format:
     //   <algorithm name>:<encrypted secret key with RSA key><encrypted data with secret key>
     pub fn encrypt(&self, data: &[u8]) -> Vec<u8> {
-        let mut rng = rand::thread_rng();
+        let mut rng = OsRng;
```

Comment:
maybe remove this `rng` variable since it is only used onced and we inline it in the others calls

Reply:
done